sdk/schema/executable.go

```diff
@@ -8,6 +8,15 @@ import (
 	"github.com/1Password/shell-plugins/sdk"
 )
 
+type SupportedCredentialAmount int
+
+const (
+	// ExactlyOne signifies that a certain executable supports exactly one credential
+	ExactlyOne SupportedCredentialAmount = iota
```

Comment:
@florisvdg Does it make sense to introduce `ExactlyOne`? If we make `SupportedCredentialAmount ` an enumeration like we are doing here, for all plugins that don't specify this field, the value taken will be this default value. Should we make this `StaticAmount`, or `FixedAmount` instead? For now, we only have plugins with exactly one credential usage, but in the future this is bound to change. Do you see any future value in having this field be more specific like `ExactlyOne` or `ExactlyTwo`?